### DISCUSSION TITLE
saw useMemo used for static array ,

### DISCUSSION THREAD

**Discussion by wassimnassour** [TanStack/table#4861] [General]
i saw useMemo used for static array, i think it's better if  we can remove the useMemo and copy the array outside of the component 

https://github.com/TanStack/table/blob/06703a56890122cedf1b2fa4b82982999537774e/examples/sorting/src/App.js#L102

**Comment by githorse** [comment]
Seems like specific case of https://github.com/TanStack/table/discussions/2642.
